doc/shared/sundials/Install.rst

```diff
@@ -2176,4 +2187,27 @@ The following command builds and installs SUNDIALS with MPI, HIP, and the Fortra
    # Need an allocation to run the tests:
    salloc -A <account> -t 10 -N 1 -p batch
    make test
-   make test_install_all
\ No newline at end of file
+   make test_install_all
+
+
+Building with SUNDIALS Addons
+-----------------------------
+
+SUNDIALS "addons" are community developed code additions for SUNDIALS that can be subsumed by the
+SUNDIALS build system so that they have full access to all internal SUNDIALS symbols. The intent is
+for SUNDIALS addons to function as if they are part of the SUNDIALS library, while allowing them to
+potentially have different licenses (although we encourage BSD-3-Clause still), code style (although
+we encourage them to follow the SUNDIALS style outlined :ref:`here <https://sundials.readthedocs.io/en/latest/developers/style_guide/index.html>`_).
```

Comment:
To make the original ``:ref:`here <Style>` `` reference work, add `'sphinx.ext.intersphinx'` to the `extensions` list in `conf.py` and add the following as well
```
intersphinx_mapping = {'sundials': (f'https://sundials.readthedocs.io/en/{doc_version}',
                                    ('../../../superbuild/build/html/objects.inv', None))}
```
When a cross reference can't be found in the build Sphinx will try to resolve it using the online version (or the local superbuild as a backup). The package guides use this to make "external" references to other packages work in the combined changelog.